build-app.sh

```diff
@@ -24,6 +24,10 @@ pip install .;
 # install dependencies that are needed to build and run the mac application
 pip install -r requirements/mac-app.txt;
 
+# work around zope.interface's namespace module shenanigans
+# see https://github.com/zopefoundation/zope.interface/issues/67
```

Comment:
@glyph Should we track a `todo` to remove this with a mimic issue linked to https://github.com/zopefoundation/zope.interface/issues/67?